LEAF_Request_Portal/utils/LEAF_importStandardConfig.php

```diff
@@ -47,6 +48,8 @@
 $db->query("delete FROM `actions`;");
 $db->query("delete FROM `records_dependencies` WHERE dependencyID=0;"); // delete invalid items -- TODO: figure out how these invalid items got written
 $db->query("TRUNCATE TABLE `step_modules`;");
+$db->query("TRUNCATE TABLE `email_templates`;");
+$db->query("TRUNCATE TABLE `events`;");

```

Comment:
If new tables are being wiped, they need to be paired with a matching import. Otherwise this would break events for the subordinate site.

Reply:
I think I just forgot to remove this. It's now gone.